### PR TITLE
Profile extended

### PR DESCRIPTION
**Pull request by jpirazusta**
Update profile feature and logout.

Flow:
gif

### CODE REVIEW

File: src/actions/userActions.js [hunk 1]
```diff
@@ -33,10 +33,13 @@ export const getProfile = createThunk('GET_PROFILE', async id => {
   return user;
 });
 
+export const updateProfile = createThunk('UPDATE_PROFILE', userService.updateProfile);
```
Comment (peter7z) [inline src/actions/userActions.js:36]: 👏

**Review by peter7z — APPROVED** [review]
Good job!

**Review by ManuViola77 — APPROVED** [review]
Excellent job!! 👏

File: src/components/SelectAvatar/index.js [hunk 2]
```diff
@@ -37,14 +39,27 @@ const SelectAvatar = ({ setAvatarData }) => {
...
     <>
-      <Image source={avatarSource || defaultProfileImage} style={styles.image} />
-      <Button handleOnPress={onSelect} title={selectAvatar} titleColor={BLACK} />
+      <View style={styles.container}>
+        <Image source={profileCircles} />
+        {avatarUrl && !avatarSource ? (
+          <Image source={{ uri: avatarUrl }} style={styles.profileImage} />
+        ) : (
+          <Image source={avatarSource || defaultProfileImage} style={styles.profileImage} />
+        )}
```
Comment (elizabethlofredo) [inline src/components/SelectAvatar/index.js]: I would suggest making a separate component for the avatar profile so you can pass as source avatarSource or avatarUrl

```suggestion
    <Avatar source={{ uri: avatarSource || avatarUrl }} />
```
for this to work you need to do `setAvatarSource(uri)` (the value not the object) on `onSelect ` function.

In this way, you send `avtarSource` if the user selects something from the library or `avatarUrl` in the other case.  Then, you can have`defaultProfileImage` as a default prop for the source on Avatar component. 

WDYT? in order to make this cleaner. You can add this on a next PR no need to change this now 😄

File: src/components/SelectAvatar/index.js [hunk 3]
```diff
@@ -37,14 +39,27 @@ const SelectAvatar = ({ setAvatarData }) => {
...
+      </View>
+      <Button
```
Comment (elizabethlofredo) [inline src/components/SelectAvatar/index.js:45]: I think the style for the text is missing here, right?

File: src/components/SelectAvatar/styles.js [hunk 4]
```diff
@@ -1,12 +1,21 @@
...
-  image: {
-    width: 80,
-    height: 80,
-    borderRadius: 40,
-    alignSelf: 'center',
-    marginVertical: 10,
+  container: {
+    alignItems: 'center',
+    marginRight: 80,
+    marginTop: 80,
```
Comment (elizabethlofredo) [inline src/components/SelectAvatar/styles.js:7]: The profile avatar should be centered on all screen dimensions, I think this is not going to work in all cases

**Review by elizabethlofredo — APPROVED** [review]
Great job! 💪

File: src/components/SelectAvatar/index.js (hunk 3 above)
Comment (jpirazusta) [reply src/components/SelectAvatar/index.js:45]: It is using the Button's style for the text. Button is a custom component, not the native one. Perhaps I could have called it 'CustomButton'.

File: src/components/SelectAvatar/styles.js (hunk 4 above)
Comment (jpirazusta) [reply src/components/SelectAvatar/styles.js:7]: The image is center aligned, but moved to the left to center the blue circle. So, it works for different screen widths. I mean, these values refer to the image dimensions, that are always the same.

**Pull request merged**